configure.ac

```
@@ -6191,6 +6191,26 @@ AS_CASE([$ac_sys_system],
   [CYGWIN*], [py_stdlib_not_available="_scproxy nis"],
   [QNX*], [py_stdlib_not_available="_scproxy nis"],
   [FreeBSD*], [py_stdlib_not_available="_scproxy spwd"],
+  [emscripten], [
```

``ac_sys_system`` for ``*-*-emscripten`` is ``Emscripten``

```suggestion
  [Emscripten], [
```

---

Yes, I just noticed that. Fixed in force push :)